frontend/public/style/_overrides.scss

```diff
@@ -109,6 +109,10 @@ h6 {
   color: var(--pf-c-button--m-plain--Color);
 }
 
+.pf-c-button.pf-c-button-icon--help-icon {
```

Comment:
If we're adding a new style instead of overriding an existing `pf-c-button-icon--help-icon` PatternFly style, I would give this class a different name with the `co-` prefix and maybe move it to a different file. @rhamilto @sg00dwin opinion?

Reply:
Agreed.  If we are overriding, we should do it with a custom class.

Reply:
`pf-c-button-icon--help-icon` is the class I made up, should I rename it and move it? If so, what should I name it and where should I move it?